.ci.yaml

```diff
@@ -3912,6 +3912,7 @@ targets:
   - name: Mac framework_tests_impeller
     recipe: flutter/flutter_drone
     timeout: 60
+    bringup: true
```

Comment:
A comment with a link to an issue will be helpful here.

Reply:
Done